src/ensemble/EnsembleBase.cpp

```diff
@@ -100,3 +106,28 @@ void Ensemble::setComponentLookUpIDs() {
 		centers += c->numLJcenters();
 	}
 }
+
+void Ensemble::setMixingrule(std::shared_ptr<MixingRuleBase> mixingrule) {
+	const int cid1 = mixingrule->getCid1();
+	const int cid2 = mixingrule->getCid2();
```

Comment:
```suggestion
	const auto [cid1, cid2] = std::minmax(mixingrule->getCid1(), mixingrule->getCid2());
```

And then simplify the following `if`s accordingly

Reply:
Done that and got rid of the if regarding the assignment to _mixingrules

Reply:
This had to be modified a bit, see [here](https://stackoverflow.com/questions/72329739/assigning-stdminmax-result-to-new-variables). But this shouldn't be a problem for the [other use](https://github.com/ls1mardyn/ls1-mardyn/blob/42bb143cc0d397916c3e626aa1b8a5b446ad94c0/src/molecules/mixingrules/MixingRuleBase.cpp#L27) of `std::minmax`, right?